docs/docs/connect/data-source/mysql.md

```diff
@@ -23,44 +23,30 @@ When connecting to MySQL, an appropriate Data Source Name (DSN) must be specifie
 
 For more details, see the [MySQL documentation on DSN formats](https://dev.mysql.com/doc/refman/8.4/en/connecting-using-uri-or-key-value-pairs.html#connecting-using-uri).
 
-<img src='/img/connect/data-sources/mysql.png' class='rounded-gif' style={{width: '75%', display: 'block', margin: '0 auto'}}/>
-<br />
 
 
-## Local credentials
+## Connect to MySQL
 
-When using Rill Developer on your local machine (i.e., `rill start`), you have the option to specify a connection string when running Rill using the `--env` flag.
-An example of passing the connection DSN to Rill via the terminal:
-
-```bash
-rill start --env connector.mysql.dsn="mysql://mysql_user:mysql_password@localhost:3306/mysql_db"
-```
-
-Alternatively, you can include the connection string directly in the source YAML definition by adding the `dsn` parameter.
-An example of a source using this approach:
+When using Rill Developer on your local machine (i.e., `rill start`), Connect to MySQL via Add Data. This will automatically create the `mysql.yaml` file in your connectors/ folder and populate the `.env` file with `connector.mysql.*` parameters depending on if you inputted parameters or connection string.
```

Comment:
1. This needs grammar & formatting polish
2. I'm not so sure we should just say "use [the] Add Data [form]". Primarily, the way to connect to MySQL is by creating the code artifact (as defined below). Secondarily, if desired, the MySQL form can assist the user to create that code.